scripts/binary/build.ts

```diff
@@ -121,7 +121,7 @@ export async function buildCypressApp (options: BuildCypressAppOpts) {
   fs.writeJsonSync(meta.distDir('package.json'), {
     ...packageJsonContents,
     scripts: {
-      postinstall: 'patch-package',
+      postinstall: 'patch-package && yarn workspace @packages/server build',
     },
   }, { spaces: 2 })
```

Comment:
add comment that this ensure a correct post-install and the native bindings are re-built for the machine per https://github.com/cypress-io/cypress/pull/26168/files#r1144897619